metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+  "dae": {
+    "release": {
+      "version": "v0.8.0",
+      "rev": "v0.8.0",
+      "hash": "sha256-Vdh5acE5i/bJ8VXOm+9OqZQbxvqv4TS/t0DDfBs/K5g=",
+      "vendorHash": "sha256-0Q+1cXUu4EH4qkGlK6BIpv4dCdtSKjb1RbLi5Xfjcew="
+    },
+    "unstable": {
+      "version": "unstable-2024-11-03.d3ab0b2",
+      "rev": "d3ab0b25e705399d7e610d6f09453623674e20d3",
+      "hash": "sha256-iUVTnFPeWQkFZBsxbnBe/lVO9FTqmQeSzDsgPa5ZqLY=",
+      "vendorHash": "sha256-rGwVsDgcNGWppte0BXQF9p0pKbnjqeNBRn5oNxrxsjU="
+    },
+    "experiment": {
+      "version": "v0.7.0",
+      "rev": "v0.7.0",
+      "hash": "sha256-9iwrwQGpryGyEUVB2reodIxuEQHkXPA4P5IYKj18elI=",
+      "vendorHash": "sha256-AtYLxR7Fw3+IOSeuPXlq4vMsnS+7MMaFANZDg0yvCl8="
+    }
```

Comment:
> Any thoughts on how we can update the metadata in `main` branch based on needs?

Here we can add a new cmd arg for specifying revision hash for `experiment`.
The interface will looks like:

```
./main.nu sync dae experiment --rev ec7cf06de4afa173a46307d5d3095f67e97a1ffd
```

Reply:
> Let's make it intuitive:

The string `prereleaz` could be any words, it will sync according the provided rev and update the `metadata` record with this key (whatever it was). See https://github.com/daeuniverse/flake.nix/pull/124#issuecomment-2459146709